src/ring_theory/int/basic.lean

```diff
@@ -86,8 +86,33 @@ end⟩
 instance : unique_factorization_monoid ℕ :=
 ⟨λ _, nat.irreducible_iff_prime⟩
 
+instance unique_units : unique (units ℕ) :=
+{ default := 1, uniq := nat.units_eq_one }
+
+instance unique_add_units : unique (add_units ℕ) :=
+{ default := 0, uniq := nat.add_units_eq_zero }
+
+@[simp] lemma normalize_eq (n : ℕ) : normalize n = n := n.mul_one
+
 end nat
 
+/-- `ℕ` is a gcd_monoid. -/
+instance : gcd_monoid ℕ :=
```

Comment:
Is it possible to shorten this even more, using `gcd_monoid_of_gcd` or `gcd_monoid_of_exists_gcd`?

Reply:
The disadvantage of this would be that `lcm` is no longer `nat.lcm` by rfl, but instead some classical.some, which is presumably at least confusing, potentially harder to work with.